.changeset/bumpy-peaches-float.md

```diff
@@ -0,0 +1,5 @@
+---
+"@assistant-ui/react": patch
```

Comment:
I kept the patch scoped to just the react package, but if there is another dependency reason this should include others let me know!